Dockerfile

```diff
@@ -127,4 +127,4 @@ ENTRYPOINT ["/sbin/docker-entrypoint.sh"]
 EXPOSE 8000
 CMD ["/usr/local/bin/supervisord", "-c", "/etc/supervisord.conf"]
 
-LABEL maintainer="Jonas Winkler <dev@jpwinkler.de>"
+LABEL maintainer="paperless-ngx team <hello@paperless-ngx.com>"
```

Comment:
While this line is being updated, maybe it should change to `LABEL org.opencontainers.image.authors=`?  It's what replaced the [previous MAINTAINER instruction](https://docs.docker.com/engine/reference/builder/#maintainer-deprecated) anyway.

There's also [other `LABEL`s which could be set](https://github.com/opencontainers/image-spec/blob/main/annotations.md#pre-defined-annotation-keys) like vendor or version, for some future work.

Reply:
Great point, I added a few of the static ones.